### ISSUE TITLE
Scaling issue when nested=output resolution unless you enable nearest-neighbor filtering (introduced with 3.11.22)

### ISSUE THREAD

**Issue by tzcrawford** [Plagman/gamescope#453]
This might not even be a bug, but I thought it was strange behavior.

Historically I have used `-w 1920 -h 1080 -W 1920 -H 1080 -b` parameters on some games like Europa Universalis 4 just to keep the mouse captive in the window/on the monitor. Upon upgrading from 3.11.21 to 3.11.22, I found that text started appearing blurry as if the game wasn't scaling correctly. I ended up finding that adding the nearest-neighbor filtering flag (`-n`) appears to makes it look much better with the 3.11.22 version.

I have seen this for multiple games, including EU4 and CS:GO, but many other games include larger or round fonts where it is less visible. I have tried this with both i3wm-gaps and cinnamon desktop, and it appears to happen in both cases on the same machine. I am using Arch Linux kernel 5.15.12-zen1-1-zen

**Comment by DadSchoorse** [comment]
Yes there was a regression, but it was already fixed in the next release. 3.11.22 is pretty old.
